pkgs/os-specific/linux/kernel/xanmod-kernels.nix

```
@@ -24,48 +24,51 @@ let
       inherit hash;
     };
 
-    structuredExtraConfig = with lib.kernel; {
-      # removed options
-      CFS_BANDWIDTH = lib.mkForce (option no);
-      RT_GROUP_SCHED = lib.mkForce (option no);
-      SCHED_AUTOGROUP = lib.mkForce (option no);
+    structuredExtraConfig =
+      with lib.kernel;
+      with (lib.kernel.whenHelpers version);
+      {
+        # removed options
```

one final thing, since I saw that `-Werror` was enforced by default

```suggestion
        # TODO: remove this once https://github.com/NixOS/nixpkgs/pull/175433 is in master
        WERROR = no;

        # removed options
```